metadata-ingestion/tests/integration/delta_lake/golden_files/local/golden_mces_table_with_string_and_array.json

```diff
@@ -0,0 +1,893 @@
+[
+{
+    "proposedSnapshot": {
+        "com.linkedin.pegasus2avro.metadata.snapshot.DatasetSnapshot": {
+            "urn": "urn:li:dataset:(urn:li:dataPlatform:delta-lake,tests/integration/delta_lake/test_data/delta_tables_nested_datatype/table_with_string_and_array,UAT)",
+            "aspects": [
+                {
+                    "com.linkedin.pegasus2avro.common.Status": {
+                        "removed": false
+                    }
+                },
+                {
+                    "com.linkedin.pegasus2avro.dataset.DatasetProperties": {
+                        "customProperties": {
+                            "partition_columns": "[]",
+                            "table_creation_time": "1708329075098",
+                            "id": "bccc302c-006d-4de9-b572-0c1939e64fc5",
+                            "version": "0",
+                            "location": "tests/integration/delta_lake/test_data/delta_tables_nested_datatype/table_with_string_and_array"
+                        },
+                        "name": "table_with_string_and_array",
+                        "tags": []
+                    }
+                },
+                {
+                    "com.linkedin.pegasus2avro.schema.SchemaMetadata": {
+                        "schemaName": "table_with_string_and_array",
+                        "platform": "urn:li:dataPlatform:delta-lake",
+                        "version": 0,
+                        "created": {
+                            "time": 0,
+                            "actor": "urn:li:corpuser:unknown"
+                        },
+                        "lastModified": {
+                            "time": 0,
+                            "actor": "urn:li:corpuser:unknown"
+                        },
+                        "hash": "",
+                        "platformSchema": {
+                            "com.linkedin.pegasus2avro.schema.OtherSchema": {
+                                "rawSchema": ""
+                            }
+                        },
+                        "fields": [
+                            {
+                                "fieldPath": "[version=2.0].[type=struct].[type=array].[type=struct].data",
+                                "nullable": false,
+                                "type": {
+                                    "type": {
+                                        "com.linkedin.pegasus2avro.schema.ArrayType": {
+                                            "nestedType": [
+                                                "record"
+                                            ]
+                                        }
+                                    }
+                                },
+                                "nativeDataType": "array<item:string>",
```

Comment:
This should be `array<string>` right ?

Reply:
if we don't keep the `fieldName i.e. item` and simply put `array<string>` then the collapsible elements are not getting generated; so for columns where we don't have the name of the `field` I am putting `item` as a name

 
![Uploading Screenshot 2024-02-26 at 12.51.05 pm.png…]()